### PR TITLE
Feature | Sort Organisation Table

### PR DESCRIPTION
**Pull request by ThatcherK**

### CODE REVIEW

**Review by edwinmp — CHANGES_REQUESTED** [review]
For validation status, a range doesn't make sense ... i.e. Critical - Success. You want to give users an option to decide what they want to see first ... so every option has to be represented. Also, if it's not a hustle, make sure that only available file statuses are shown. Lastly, don't mix up cases ... Upper, lower - stick to one - prefer to make every first letter upper case, except for connecting words like `in` or `that` (whatever they're called in english)  i.e `Validation Status: Critical` ... notice the use of spaces.

As for the counters - please change the label text for the error counts to be in the matching colour for that error ... i.e Critical is purple. Only the labels, not the number.

On mobile, have each the feedback and the sort dropdown each on its own row.

image

**Comment by edwinmp** [comment]
As I mentioned, ONLY on mobile
image

Again, just on mobile, add padding or a margin between the two items. `mt-2` should do.

Replace `Alphabetical` with `A - Z` and `Z - A`
Replace `New-Old` with `Newest`, `Old-New` with `Oldest`

The default sorting criteria should be showing by default. You'll need to add a label to the field that clearly identifies its purpose as a sorting dropdown.

Change the drop-down colours to IATI colours.

**Comment by ThatcherK** [comment]
@edwinmp  this is ready for another review

File: src/pages/OrganisationPage.vue [hunk 1]
```diff
@@ -92,3 +125,19 @@
...
+  .multiselect__option--selected.multiselect__option--highlight {
+    background: #81c3d6;
+  }
+  .multiselect__option--selected.multiselect__option--highlight:after {
+    background: #81c3d6;
+  }
+  .multiselect__option--group-selected.multiselect__option--highlight {
+    background: #81c3d6;
+  }
+</style>
```
Comment (edwinmp) [inline src/pages/OrganisationPage.vue:143]: What if I need this used else where (very likely), do I have to copy paste all this for every usage?
Create a component that embodies this customisation so we can use it throughout https://vuejs.org/guide/components/attrs.html#nested-component-inheritance

File: src/utils/document.js [hunk 2]
```diff
@@ -339,3 +339,122 @@ export const getFeedbackCategoryLabel = (category) => {
   };
   return categories[category];
 };
+
+export const processedTableDocumentFields = (documents, sortKey, sortDirection) => {
+  if (documents.length) {
+    // console.log(sortDirection);
```
Comment (edwinmp) [inline src/utils/document.js:345]: Remove

File: src/utils/document.js [hunk 3]
```diff
@@ -339,3 +339,122 @@ export const getFeedbackCategoryLabel = (category) => {
   };
   return categories[category];
 };
+
+export const processedTableDocumentFields = (documents, sortKey, sortDirection) => {
```
Comment (edwinmp) [inline src/utils/document.js:343]: Generic naming ... Don't need to know this is being used on a table. If it's a sort function, name it so. Current naming tells me that this function is returning processed table document fields.

**Review by edwinmp — CHANGES_REQUESTED** [review]

**Comment by edwinmp** [comment]
Fix the conflicts.

**Comment by ThatcherK** [comment]
conflicts fixed

**Pull request merged**